diff -urN squashfs4.old/squashfs-tools/mksquashfs.c squashfs4.dev/squashfs-tools/mksquashfs.c
--- squashfs4.old/squashfs-tools/mksquashfs.c	2009-04-05 21:22:48.000000000 +0000
+++ squashfs4.dev/squashfs-tools/mksquashfs.c	2026-08-23 00:00:00.000000000 +0300
@@ -30,6 +30,7 @@
 #include <unistd.h>
 #include <stdio.h>
 #include <sys/time.h>
 #include <sys/types.h>
+#include <sys/sysmacros.h>
 #include <sys/stat.h>
 #include <fcntl.h>
 #include <errno.h>
diff -urN squashfs4.old/squashfs-tools/unsquashfs.c squashfs4.dev/squashfs-tools/unsquashfs.c
--- squashfs4.old/squashfs-tools/unsquashfs.c	2009-04-05 21:23:06.000000000 +0000
+++ squashfs4.dev/squashfs-tools/unsquashfs.c	2026-08-23 00:00:00.000000000 +0300
@@ -21,7 +21,8 @@
 * unsquashfs.c
 */
 
 #include "unsquashfs.h"
+#include <sys/sysmacros.h>
 #include "squashfs_swap.h"
 #include "squashfs_compat.h"
 #include "read_fs.h"
